backend/src/main/java/com/storageroom/security/config/SecurityConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package com.storageroom.security.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+    private DataSource dataSource;
+
+    @Autowired
+    public SecurityConfiguration(DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+    @Override
+    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+        auth.jdbcAuthentication()
+                .withDefaultSchema()
+                .dataSource(dataSource)
+                .withUser("user")
+                .password("{bcrypt}" + new BCryptPasswordEncoder().encode("password"))
```

Comment:
Dlaczego jest to `{bcrypt}`?

Reply:
no bo bcrypt bezpieczny i chyba teraz jest zalecany

Reply:
to rozumiem, ale nie czaje po co dodajesz ten string `{bcrypt}`

Reply:
sam ten string "{bcrypt}" to sol

Reply:
https://spring.io/blog/2017/11/01/spring-security-5-0-0-rc1-released#password-storage-format

potrzebne do password encodera